README.md

```diff
@@ -148,13 +147,50 @@ To run this on the sample app:
 ```
 
 2. Try running the following command:
-
 ```
  ./gradlew runAffectedUnitTests -Paffected_module_detector.enable
 ```
 
 You should see zero tests run. Make a change within one of the modules and commit it. Rerunning the command should execute tests in that module and its dependent modules.
 
+## Custom tasks 
+
+If you want to add a custom gradle command to execute with impact analysis 
+you must create enum which implementing [AffectedModuleTaskType](). 
```

Comment:
note: a link will be added after merging this MR.

Reply:
Also can you please explain why you have to use an enum? Is this not valid as well

```
class DetektTask : AffectedModuleTaskType {
    override val commandByImpact = "runDetektByImpact"
    override val originalGradleCommand = "detekt"
    override val taskDescription = "Run static analysis tool without auto-correction by Impact analysis"    
}
```

Reply:
Thank you for the review! 
Yep, you are right, users can use other way for implementing the interface. I changed doc.

Reply:
To make something for the sample you can just deploy to local maven and then when you use the sample app it should pull from there (ex: `./gradlew :affectedmoduledetector:publishToMavenLocal`)

It would be great if you could post a sample. I tried to copy this with putting `CustomImpactAnalysisTaskType` in the root of my project and then referencing it with `customTasks = [ CustomImpactAnalysisTaskType.DETEKT_TASK ]` and I get a build failure. If I move `CustomImpactAnalysisTaskType` into `buildSrc` it works as expected but at that point its probably just as easy to create your custom task using the `buildSrc` directory

Reply:
@joshafeinberg I thought about your words and decided to rethink the approach towards simplification. I made a CustomTask class which is now available without any dependencies.
Now it's easy to use it to specify custom tasks directly from the configuration file.

 * `build.gradle
```
     
      affectedModuleDetector {
           ...
           customTasks = [ // <- list of custom gradle invokes
               new AffectedModuleConfiguration.CustomTask(
                    "runSomeCustomTaskByImpact",
                    "someTaskForExample",
                    "Task description."
                )
           ]
           ...
      }
     `
```

You can see it in sample. 
What do you think about this approach?